velox/common/memory/MappedMemory.h

```diff
@@ -418,6 +418,16 @@ class MappedMemory : public std::enable_shared_from_this<MappedMemory> {
       uint64_t size,
       uint64_t maxMallocSize = kMaxMallocBytes) noexcept;
 
+  // Reserves memory of page size that is allocated elsewhere externally. No
```

Comment:
// Reserves memory in page unit that is allocated externally. ... Only the internal memory usage counters are increased according to the reservation request. ...